### PR TITLE
Grid Toast Utils

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,
I have added the Grid and Toast page to the documentation site.
For Utils:
- Text utils are added to the Typography page
- Color utils are added to the Colors page

### CODE REVIEW

**Review by aayushjain-code — APPROVED** [review]
Footer is overlapping with upper items @ApurvaSawant11 Try to fix them

File: css/colors.css [hunk 1]
```diff
@@ -45,3 +47,15 @@
 .palette-container > .color:nth-child(8){
     background: var(--dark-gray);
 }
+
+.color-utils-class{
+    font-size: 1rem;
+    color: var(--secondary);
+    background: var(--white-color);
```
Comment (MayankKumar10) [inline css/colors.css:54]: you can give another name to white color

**Review by MayankKumar10 — APPROVED** [review]
Everything else LGTM you can merge it

**Review by ArjunJunna — APPROVED** [review]
Code is clean. Looks fine to me.

**Comment by ApurvaSawant11** [comment]
> Footer is overlapping with upper items @ApurvaSawant11 Try to fix them

Fixed it. Thanks

File: css/colors.css (hunk 1 above)
Comment (ApurvaSawant11) [reply css/colors.css:54]: Okay, will try to name it something else in next PR

**Pull request merged**